sdk/schemaregistry/Azure.Data.SchemaRegistry/README.md

```diff
@@ -0,0 +1,104 @@
+# README.md template
```

Comment:
Just a reminder that you will need to fill this out for SR.

Reply:
In progress.

Reply:
Updated.